inc/functions.php

```diff
@@ -33,6 +33,51 @@ function rwmb_meta( $key, $args = array(), $post_id = null ) {
 	}
 }
 
+if ( ! function_exists( 'rwmb_set_meta' ) ) {
+	/**
+	 * Set post meta.
+	 *
+	 * @param string   $key     Meta key. Required.
+	 * @param string   $value   Meta value. Required.
+	 * @param array    $args    Array of arguments. Optional.
+	 * @param int|null $post_id Post ID. null for current post.
+	 *
+	 * @return mixed
+	 */
+	function rwmb_set_meta( $key, $value, $args = array(), $post_id ) {
+		$args  = wp_parse_args( $args );
+		$field = rwmb_get_field_settings( $key, $args, $post_id );
+		$type = get_post_type( $post_id );
+
+		if ( false === $field ) {
+			$args  = wp_parse_args(
+				$args,
+				array(
+					'type'     => 'text',
+					'multiple' => false,
+					'clone'    => false,
+				)
+			);
```

Comment:
Is this necessary? Can I put the default values in the first line of the function?